### PR TITLE
Added an initial version of a texton module that represents the appea…

### PR DESCRIPTION
**Pull request by guidoAI**
…rance of an image with a texton distribution. In past work, we have used texton distributions for instance for obstacle detection. If the entropy of the texton distribution becomes low, this is typically an indication of a close obstacle. This "appearance variation cue" can be a good complement to using optical flow measurements for obstacle avoidance. There are may other uses for texton distributions though, while they are computationally quite cheap.

### CODE REVIEW

**Comment by gautierhattenberger** [comment]
please fix indentation or use the `fix_code_style.s` script.
also there are some comments that can be removed (e.g. in module xml file).

File: sw/airborne/modules/computer_vision/textons.c [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+uint8_t dictionary_number = 0;
+
+// status variables
+uint8_t dictionary_ready = 0;
+float alpha = 0.0;
+
+// File pointer for saving the dictionary
+static FILE *dictionary_logger = NULL;
+#ifndef DICTIONARY_PATH
+#define DICTIONARY_PATH /data/video/
```
Comment (flixr) [inline sw/airborne/modules/computer_vision/textons.c]: should be added to the doc section in the module xml file...

**Comment by guidoAI** [comment]
I fixed the code style and added the documentation.

**Comment by flixr** [comment]
@guidoAI thx!
If you tested this and it compiles/works it looks good to me to merge...

**Pull request merged**